crates/tauri-bundler/src/error.rs

```diff
@@ -3,19 +3,41 @@
 // SPDX-License-Identifier: Apache-2.0
 // SPDX-License-Identifier: MIT
 
-use std::{io, num, path};
+use std::{
+  fmt::Display,
+  io, num,
+  path::{self, PathBuf},
+};
 use thiserror::Error as DeriveError;
 
 /// Errors returned by the bundler.
 #[derive(Debug, DeriveError)]
 #[non_exhaustive]
 pub enum Error {
+  /// Error with context. Created by the [`Context`] trait.
+  #[error("{0}: {1}")]
+  Context(String, Box<Self>),
+  /// File system error.
+  #[error("{context} {path}: {error}")]
+  Fs {
+    /// Context of the error.
+    context: &'static str,
+    /// Path that was accessed.
+    path: PathBuf,
+    /// Error that occurred.
+    error: io::Error,
+  },
+  /// Child process error.
+  #[error("failed to run command {command}: {error}")]
+  CommandFailed {
+    /// Command that failed.
+    command: String,
+    /// Error that occurred.
+    error: io::Error,
+  },
   /// Error running tauri_utils API.
   #[error("{0}")]
   Resource(#[from] tauri_utils::Error),
-  /// Bundler error.
-  #[error("{0:#}")]
-  BundlerError(#[from] anyhow::Error),
```

Comment:
Can we revert this and add a todo and change the docs saying that this variant is unused now? There are projects using tauri-bundler and this is a breaking api change.